x-pack/plugins/ml/public/application/datavisualizer/stats_datagrid/components/field_data_expanded_row/number_content.tsx

```diff
@@ -0,0 +1,173 @@
+import {
+  MetricDistributionChart,
+  MetricDistributionChartData,
+  buildChartDataFromStats,
+} from '../../../index_based/components/field_data_card/metric_distribution_chart';
+import { TopValues } from '../../../index_based/components/field_data_card/top_values';
+import { ExpandedRowFieldHeader } from '../expanded_row_field_header';
+import { VerticalSpacer } from '../vertical_spacer';
+
+const METRIC_DISTRIBUTION_CHART_WIDTH = 325;
+const METRIC_DISTRIBUTION_CHART_HEIGHT = 200;
+
+interface SummaryTableItem {
+  function: string;
+  display: ReactNode;
+  value: number | string | undefined | null;
+}
+
+export const NumberContent: FC<FieldDataCardProps> = ({ config }) => {
+  const { stats, fieldFormat } = config;
+
+  useEffect(() => {
+    const chartData = buildChartDataFromStats(stats, METRIC_DISTRIBUTION_CHART_WIDTH);
+    setDistributionChartData(chartData);
+  }, []);
+
+  const defaultChartData: MetricDistributionChartData[] = [];
+  const [distributionChartData, setDistributionChartData] = useState(defaultChartData);
+
+  if (stats === undefined) return null;
+  const { min, median, max, distribution } = stats;
+
+  const summaryTableItems = [
+    {
+      function: 'min',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.minLabel"
+          defaultMessage="min"
+        />
+      ),
+      value: kibanaFieldFormat(min, fieldFormat),
+    },
+    {
+      function: 'median',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.medianLabel"
+          defaultMessage="median"
+        />
+      ),
+      value: kibanaFieldFormat(median, fieldFormat),
+    },
+    {
+      function: 'max',
+      display: (
+        <FormattedMessage
+          id="xpack.ml.fieldDataCardExpandedRow.numberContent.maxLabel"
+          defaultMessage="max"
+        />
+      ),
+      value: kibanaFieldFormat(max, fieldFormat),
+    },
+  ];
+  const summaryTableColumns = [
+    {
+      name: '',
+      render: (summaryItem: { display: ReactNode }) => summaryItem.display,
+      width: '75px',
+    },
+    {
+      field: 'value',
+      name: '',
+      render: (v: string) => <strong>{v}</strong>,
+    },
+  ];
+
+  const summaryTableTitle = i18n.translate(
+    'xpack.ml.fieldDataCardExpandedRow.numberContent.summaryTableTitle',
+    {
+      defaultMessage: 'Summary',
+    }
+  );
+  return (
+    <EuiFlexGroup direction={'row'} data-test-subj={'mlNumberSummaryTable'}>
+      <EuiFlexItem>
+        <ExpandedRowFieldHeader>{summaryTableTitle}</ExpandedRowFieldHeader>
+        <EuiBasicTable<SummaryTableItem>
+          className={'mlDataVisualizerSummaryTable'}
+          compressed
+          items={summaryTableItems}
+          columns={summaryTableColumns}
+          tableCaption={summaryTableTitle}
+        />
+      </EuiFlexItem>
+      <VerticalSpacer />
```

Comment:
Do you actually need this component? 🤔 Have you tried setting a gutter size instead?

Reply:
Agree... `gutterSize='xl'` should work

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/4ea6d174728033f29d81392da47db1f3b409d4a5